utils/hooks/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Make sure we are inside the repository.
+cd "${BASH_SOURCE%/*}" &&
+cd ../..
+
+scripts_folder=$(pwd)'/utils/'
+hook_scripts=("check_tags.sh" "check_cross_ref_images.sh")
+
+# Look for the added or modified Markdown files
+files=($(git diff-index --diff-filter=AM --cached HEAD --name-only -- '*.md'))
+
+# Loop over the scripts if the array is not empty
```

Comment:
Is this inspired from something you have seen on the web? Just curious; seems all right.

Reply:
Inspired as a result of a research on the web, yes.

If another alternative would happen to be more robust, please go ahead.